src/utilities/util_dump.c

```diff
@@ -779,6 +779,12 @@ dump_record(
     WT_ASSERT((WT_SESSION_IMPL *)session, key != NULL);
 
     current_key = key;
+    if (json) {
+        char json_key[] = "\"key0\" : \"";
+        strcat(json_key, current_key);
```

Comment:
WiredTiger has it's own library of doing string concats and buffer formats. I would ideally use something like this:
```
        WT_ERR(__wt_buf_set(session_impl, tmp, "", 0));
        if (checkpoint != NULL)
            WT_ERR(__wt_buf_catfmt(session_impl, tmp, "checkpoint=%s,", checkpoint));
```
(Inside the same file as reference)

Reply:
Changed in most recent commit